grafana-plugin/src/containers/Rotations/ScheduleFinal.tsx

```diff
@@ -62,10 +61,6 @@ const _ScheduleFinal: FC<ScheduleFinalProps> = observer(
       onShowOverrideForm('new', shiftStart, shiftEnd);
     };
 
-    useEffect(() => {
-      refreshEvents(scheduleId);
-    }, [selectedTimezoneOffset]);
```

Comment:
so this was refetching events if you changed the timezone dropdown and this will no longer be the case? Is there any negative consequences of that?

Reply:
Since your comment I've added refreshing schedules events on TimezoneSelect onChange callback, it settles things